storage/influxdb/influxdb.go

```diff
@@ -31,39 +33,44 @@ func init() {
 }
 
 type influxdbStorage struct {
-	client         *influxdb.Client
-	machineName    string
-	tableName      string
-	bufferDuration time.Duration
-	lastWrite      time.Time
-	series         []*influxdb.Series
-	lock           sync.Mutex
-	readyToFlush   func() bool
+	client          *influxdb.Client
+	machineName     string
+	database        string
+	retentionPolicy string
+	bufferDuration  time.Duration
+	lastWrite       time.Time
+	points          []*influxdb.Point
+	lock            sync.Mutex
+	readyToFlush    func() bool
 }
 
+// Series names
 const (
-	colTimestamp          string = "time"
-	colMachineName        string = "machine"
-	colContainerName      string = "container_name"
-	colCpuCumulativeUsage string = "cpu_cumulative_usage"
+	// Cumulative CPU usage
+	serCpuUsageTotal  string = "cpu_usage_total"
```

Comment:
Using `-` may mean users have to quote everything in their queries, otherwise might be seen as operator.